vector/src/main/java/im/vector/app/features/home/room/detail/TimelineFragment.kt

```diff
@@ -1947,14 +1930,11 @@ class TimelineFragment @Inject constructor(
                     timelineViewModel.handle(RoomDetailAction.TapOnFailedToDecrypt(informationData.eventId))
                 }
             }
-            else                                 -> {
-                onThreadSummaryClicked(informationData.eventId, isRootThreadEvent)
-            }
             is MessageLocationContent            -> {
                 handleShowLocationPreview(messageContent, informationData.senderId)
             }
             else                                 -> {
-                Timber.d("No click action defined for this message content")
+                onThreadSummaryClicked(informationData.eventId, isRootThreadEvent)
```

Comment:
This change is not ideal (but I know it's due to a recent merge), I would prefer to read:
```kotlin
if (isRootThreadEvent) {
    onThreadSummaryClicked(informationData.eventId)
} else {
    Timber.d("No click action defined for this message content")
}
```